app/assets/stylesheets/va_crosswalks.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the VaCrosswalks controller here.
```

Comment:
Yup - although I've been leaving them in to the end, on the off-chance I will need them at some point - but that's unlikely... I'll delete